tests/unit/utils/managers/test_context_manager.py

```diff
@@ -208,3 +211,37 @@ def has_table(table_name):
         # Then
         mock.assert_called_once_with('table_name', 'new_table_name')
         assert result == 'new_table_name'
+
+    def test_list_tables(self, mocker):
+        # Given
+        Dataset = namedtuple('Dataset', ['name', 'updated_at'])
+
+        mocker.patch('cartoframes.io.managers.context_manager._create_auth_client')
+        mocker.patch.object(ContextManager, 'execute_query', return_value={
```

Comment:
We could remove `execute_query` mocks right?

Reply:
Ups!